blocks/blog-topics/blog-topics.js

```diff
@@ -0,0 +1,23 @@
+import ffetch from '../../scripts/ffetch.js';
+import { BLOG_QUERY_INDEX, getLocale } from '../../scripts/scripts.js';
+import { p, a } from '../../scripts/dom-helpers.js';
+
+async function getLocaleTopics() {
+  const locale = getLocale();
+  window.sidebarTopics = ffetch(`${BLOG_QUERY_INDEX}`)
```

Comment:
minor, not blocking:
* You don't have to save the topics on the window object, as they are only being accessed 1 time per page.
* `BLOG_QUERY_INDEX` is already a string, so you can call directly `ffetch(BLOG_QUERY_INDEX)`